ReactiveUI/ObservableAsPropertyHelper.cs

```diff
@@ -24,6 +25,7 @@ public sealed class ObservableAsPropertyHelper<T> : IHandleObservableErrors, IDi
         readonly IConnectableObservable<T> _source;
         IDisposable _inner;
                 
+        private int _activated;
```

Comment:
@flagbug that'd be because `Interlocked` is being used for thread-safe compare/exchange, and `Interlocked` does not support `bool`.